README.md

```diff
@@ -410,7 +410,19 @@ mutation($id: ID!) {
   { id: 5 }
 );
 ```
+### use urql without using components
```

Comment:
I think it's better to change the sentence case to this: "Use Urql without Using Components" (Use title.sh if you're not a native English speaker like me)

Reply:
I didn't capitalize because the previous titles are not capitalized either (mutation, query etc).

Reply:
Those are the API ref, this is a title. 

Look at these: 
> What is urql
> Why this exists
> How its different (typo here tho! @kenwheeler)
> Install
> Getting Started